### PR TITLE
simplified tjm data check process, renamed object properties, removed port/ http check

### PR DESCRIPTION
**Pull request by ciorg**
simplified tjm data check process for jobs files and removed redundant code
also renamed object properties to be consistent with other teraslice code and changed names to be more descriptive
removed the port/ http check since the port is not required and http will go to https at some point
addresses issues #24 and #28

### CODE REVIEW

File: cmds/asset.js [hunk 1]
```diff
@@ -123,8 +128,7 @@ exports.handler = (argv, _testTjmFunctions) => {
             })
             .catch(err => reply.fatal((err.message)));
     } else if (argv.cmd === 'status') {
-        const assetName = assetJson.name;
-        return Promise.each(cluster => latestAssetVersion(cluster, assetName));
+        clusters.forEach(cluster => latestAssetVersion(cluster))
```
Comment (peterdemartini) [inline cmds/asset.js]: We should return a Promise here.

File: cmds/reset.js [hunk 2]
```diff
@@ -3,20 +3,17 @@
...
-    const jsonData = require('./cmd_functions/json_data_functions')();
-    const jobData = jsonData.jobFileHandler(argv.jobFile);
-    const jobContents = jobData[1];
-    const jobFilePath = jobData[0];
+    require('./cmd_functions/json_data_functions')(argv).returnJobData(true);
 
-    delete jobContents.tjm;
-    return fs.writeJson(jobFilePath, jobContents, { spaces: 4 })
-        .then(() => reply.green(`TJM data was removed from ${argv.jobFile}`))
+    delete argv.job_file_content.tjm;
```
Comment (peterdemartini) [inline cmds/reset.js]: We should not be mutating the `argv` object.

**Comment by ciorg** [comment]
I changed jobFile to job_file because it becomes a property of the argv object, and other object property names use snake case.  Admittedly, it does look a bit weird.   Trying to make everything consistent.

**Comment by ciorg** [comment]
I pushed some changes and it's ready for another look

**Comment by ciorg** [comment]
changed tjmObject to tjmConfig and bumped the package.json version to v1.1.1, could not find where argv is being passed to cmd_functions/function.js.  Agree that it should always be tjmConfig

**Pull request merged**